src/main/kotlin/org/jetbrains/intellij/IntelliJPlugin.kt

```diff
@@ -113,22 +116,32 @@ open class IntelliJPlugin : Plugin<Project> {
     }
 
     private fun checkGradleVersion(project: Project) {
+        if (project.isBuildFeatureDisabled(BUILD_FEATURE_CHECK_GRADLE_VERSION)) {
+            return
+        }
         if (Version.parse(project.gradle.gradleVersion) < Version.parse("6.7")) {
             throw PluginInstantiationException("${IntelliJPluginConstants.NAME} requires Gradle 6.7 and higher")
         }
     }
 
     private fun checkPluginVersion(project: Project) {
+        if (project.isBuildFeatureDisabled(BUILD_FEATURE_CHECK_GRADLE_INTELLIJ_PLUGIN_VERSION)) {
+            return
```

Comment:
IMHO it would be safer to disable this check by default (for now). needs more research about caveats